ui/perfherder/alerts/Pagination.jsx

```diff
@@ -16,6 +16,15 @@ class PaginationGroup extends React.Component {
          aren't working correctly (icons aren't visible)
          so they haven't been added */
       <Pagination aria-label={`Page ${page}`}>
+        {pageNums[0] > 1 && (
```

Comment:
`pageNums` feels ambiguous. Is it referring to all available pages out there or only to the selectable ones?
We should rename this to something clearer.

Reply:
It is referring to selectable ones, but it is from props. If we rename here, the prop name will remain the same, and I am worried it will create more confusion :)
`paginationPageNums` sounds specific enough to you? `viewablePageNums`?

Reply:
> [...] If we rename here, the prop name will remain the same, and I am worried it will create more confusion :)

If need be, we can rename that one too. After all, that's an even more public interface.
If we can make it more concise & readable, let's do it.

> `paginationPageNums` sounds specific enough to you? `viewablePageNums`?

I think the 2nd alternative is better.